Mystter_SendTweet/Helpers/MessageHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Mystter_SendTweet.Languages;
+using System;
+using System.Windows.Forms;
+
+namespace Mystter_SendTweet.Helpers {
+  public class MessageHelper {
+    public static void Show(string message) {
+      MessageBox.Show(message, Information.Title);
+    }
+
+    public static void Show(params string[] messages) {
+      string message = string.Join(Environment.NewLine, messages);
+      Show(message);
+    }
+
+    public static bool RetryAddingAccount() {
+      return ShowYesNo(Resources.yetAdded1 + Environment.NewLine + Resources.yetAdded2);
```

Comment:
Not important,, but `ShowYesNo(Resources.yetAdded1, Resources.yetAdded2);` is better.